### PR TITLE
Closes #2: [FEATURE] - Add issue templates

### LINKED ISSUE #2: [FEATURE] - Add issue templates
Add issue templates to the repository.

### PR DESCRIPTION
**Pull request by heiko-holz**
Closes #2

### CODE REVIEW

**Comment by EverSnaxolotl** [comment]
Looks good to me!

**Review by EverSnaxolotl — CHANGES_REQUESTED** [review]
Wanted to see what happens if I do this - please change something

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
```
Comment (EverSnaxolotl) [inline .github/ISSUE_TEMPLATE/feature_request.md:20]:
```suggestion
Add any other context or a screenshot about the feature request here.
```

**Review by EverSnaxolotl — APPROVED** [review]
Good :)

**Review by ObviZelin — APPROVED** [review]
I agree I agree

**Pull request merged**